src/core/transform.ts

```diff
@@ -136,7 +136,9 @@ export async function transform(
       infos
         .forEach(({ as, name }) => {
           if (as) {
-            if (name === '*')
+            if (!name)
```

Comment:
```suggestion
            if (name === 'default')
```

We should use `default` to be explict.